include/aie/Targets/AIETargets.h

```diff
@@ -61,8 +61,8 @@ AIETranslateToCDODirect(mlir::ModuleOp m, llvm::StringRef workDirPath,
 mlir::LogicalResult
 AIETranslateToTxn(mlir::ModuleOp m, llvm::raw_ostream &output,
                   llvm::StringRef workDirPath, bool outputBinary = false,
-                  bool aieSim = false, bool xaieDebug = false,
-                  bool enableCores = true);
+                  bool outputCtrlpkt = false, bool aieSim = false,
```

Comment:
A new function e.g. `AIETranslateToControlPackets` would be better than overloading the existing function with a third output format that is not "`ToTxn`"

Reply:
Agreed. A new function is created to avoid overloading the txn workflow. It still calls the same `translateToTxn` though, as internally we still translate the device op to transaction binary, before it gets further translated into control packet ops.